### ISSUE TITLE
package.json scripts should use double quotes

### ISSUE THREAD

**Issue by XhmikosR** [npm/template-oss#63]
Another thing I noticed in some npm projects is the use of `'` in the npm scripts, for example: https://github.com/npm/template-oss/blob/ecee70cbd59541c580f76720cc2dfefb1f417603/package.json#L11

This won't work in all shells and can cause issues. Using double quotes will work everywhere.

I should probably make a new issue about this, I'm just not sure if this is handled here.

EDIT, NVM I just saw it's handled here :) https://github.com/npm/template-oss/blob/ecee70cbd59541c580f76720cc2dfefb1f417603/lib/postinstall/update-package.js#L12

_Originally posted by @XhmikosR in https://github.com/npm/template-oss/issues/62#issuecomment-1051898813_

**Comment by XhmikosR** [comment]
This is what happens with single quotes on Windows 10 cmd:

```
C:\Users\xmr\Desktop\gauge>npm run lint

> gauge@4.0.2 lint
> eslint '**/*.js'


Oops! Something went wrong! :(

ESLint: 8.10.0

No files matching the pattern "'**/*.js'" were found.
Please check for typing mistakes in the pattern.
```

Using double quotes will work everywhere.

**Comment by lukekarrys** [comment]
closed by #79
